client/dashboard/emails/choose-email-solution/index.tsx

```diff
@@ -1,25 +1,251 @@
-import { useRouter } from '@tanstack/react-router';
-import { __ } from '@wordpress/i18n';
+import { GoogleEmailSubscription, TitanEmailSubscription } from '@automattic/api-core';
+import { useNavigate } from '@tanstack/react-router';
+import {
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+	Button,
+	Icon,
+} from '@wordpress/components';
+import { sprintf, __ } from '@wordpress/i18n';
+import { wordpress } from '@wordpress/icons';
+import { useState, useEffect } from 'react';
+import { useAuth } from '../../app/auth';
+import { addProfessionalEmailRoute, addGoogleMailboxRoute } from '../../app/router/emails';
 import { PageHeader } from '../../components/page-header';
 import PageLayout from '../../components/page-layout';
+import { PriceDisplay } from '../../components/price-display';
 import { Text } from '../../components/text';
+import GoogleLogo from '../../images/google-logo.svg';
+import {
+	hasEmailForwards,
+	hasGSuiteWithUs,
+	hasTitanMailWithUs,
+	isGoogleWorkspaceSupportedDomain,
+} from '../../utils/domain';
+import { EmailNonDomainOwnerNotice } from '../components/email-non-domain-owner-notice';
+import { useAnnualSavings } from '../hooks/use-annual-savings';
+import { useDomainFromUrlParam } from '../hooks/use-domain-from-url-param';
+import { useEmailProduct } from '../hooks/use-email-product';
+import poweredByTitanLogo from '../resources/powered-by-titan-caps.svg';
+import { IntervalLength } from '../types';
+import { isDomainEligibleForTitanIntroductoryOffer } from '../utils/is-domain-eligible-for-titan-introductory-offer';
+import { isMonthlyEmailProduct } from '../utils/is-monthly-email-product';
+import { ExistingForwardsNotice } from './components/existing-forwards-notice';
+
+import './style.scss';
 
 export default function ChooseEmailSolution() {
-	const router = useRouter();
-	// Extract params from the current match for this route
-	const match = router.state.matches[ router.state.matches.length - 1 ];
-	const params = ( match?.params ?? {} ) as { domain?: string; type?: string };
-	const { domain = '' } = params;
+	const { domain, site } = useDomainFromUrlParam();
+
+	const [ billingInterval, setBillingInterval ] = useState( 'annually' as IntervalLength );
```

Comment:
```suggestion
	const [ billingInterval, setBillingInterval ] = useState< IntervalLength >( 'annually' );
```

Reply:
Nice, thanks! Done in https://github.com/Automattic/wp-calypso/pull/106555/commits/beabb99a780623fc8a58c343ee96b7bd6a2c91df